internal/xds/translator/testdata/out/ratelimit-config/empty-header-matches.yaml

```diff
@@ -1,13 +1,19 @@
 domain: first-listener
 descriptors:
-  - key: first-route-key-rule-0-match--1
-    value: first-route-value-rule-0-match--1
-    rate_limit:
-      requests_per_unit: 5
-      unit: SECOND
-      unlimited: false
-      name: ""
-      replaces: []
-    descriptors: []
+  - key: key-first-route
```

Comment:
slightly prefer just `first-route`, no `key` or `value` prefix.

Reply:
+1, it's better to remove redundant `key` and `value` prefix, since the descriptor key and value are already speaking for itself.